MonoGame.Framework/GraphicsDeviceManager.cs

```diff
@@ -171,12 +171,13 @@ protected void OnDeviceDisposing(EventArgs e)
             Raise(DeviceDisposing, e);
         }
 
-        protected void OnDeviceResetting(EventArgs e)
+        // TODO these should be made protected, but currently WP calls these in SurfaceUpdateHandler
```

Comment:
> Find issues (here on GitHub) related to GraphicsDeviceManager or come up with things that might be issues

One issue I found recently is https://github.com/MonoGame/MonoGame/issues/5173.

Also WP8.0 with `DrawingSurfaceBackgroundGrid` fails to initialize because now `BackBufferWidth`/`BackBufferHeight` is set to zero. One solution is to overwrite them with values from `WindowsPhoneGameWindow.Width`/`WindowsPhoneGameWindow.Heigth`. 
https://github.com/nkast/MonoGame/blob/7ad344d3eeb378bbd7429026f59c8857fea96e70/MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs#L226-L228

There are a couple of other issues with WP8's GraphicsDevice but never made a proper PR, I figure they most probably be noise until the platform is dropped. Anyways, FYI: 

This is wrong. `DrawingSurface`, the default in templates, support both Portrait/Landscape. https://github.com/MonoGame/MonoGame/issues/2332
https://github.com/nkast/MonoGame/blob/7ad344d3eeb378bbd7429026f59c8857fea96e70/MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs#L223-L224 
I would say it's also unnecessary for `DrawingSurfaceBackgroundGrid` as well.(https://github.com/MonoGame/MonoGame/pull/2900) 

This code crashes when used with `DrawingSurfaceBackgroundGrid`.
https://github.com/MonoGame/MonoGame/blob/41f0761e1dff502e64ad02aa6bdb5d0839d8c8b2/MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs#L158-L163
Something that works is 
https://github.com/nkast/MonoGame/blob/7ad344d3eeb378bbd7429026f59c8857fea96e70/MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs#L158-L165